vision/unit_tests/test_coordinate_lengths.py

```diff
@@ -0,0 +1,151 @@
+"""
+Testing vision.deskew.coordinate_lengths.py
+"""
+
+import unittest
+
+import numpy as np
+
+from vision.deskew.coordinate_lengths import latitude_length
+from vision.deskew.coordinate_lengths import longitude_length
+
+
+class TestCoordinates(unittest.TestCase):
+    """
+    Testing the coordinate lengths module
+    """
```

Comment:
I'd like to see a few additional things added for each of your tests:
- In the tests that you already have, you should add some `assertIsInstance` checks for whether the parameters that you are passing are the correct type as specified by the function you are testing. Also, use `assertIsInstance` to check whether the return value you are getting from the function is the correct type. Here's some more info on `assertIsInstance`: [https://www.geeksforgeeks.org/python-unittest-assertisinstance-function/](https://www.geeksforgeeks.org/python-unittest-assertisinstance-function/)